.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,71 @@
+name: "Request a Feature"
+description: Request a change for SFML
+labels: ["feature"]
+body:
+    - type: markdown
+      attributes:
+          value: |
+              Thank you for bringing up this idea for discussion! 🙂
+
+    - type: checkboxes
+      attributes:
+          label: Prerequisite Checklist
+          description: "Before you submit the issue, we'd like you to consider the follow points."
+          options:
+              - label: "I searched for [existing issues](https://github.com/search?q=repo%3ASFML%2FSFML&type=issues) to prevent duplicates"
+                required: true
+              - label: "I searched for [existing discussions on the forum](https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fen.sfml-dev.org) to prevent duplicates"
+                required: true
+              - label: "I understand [SFML's scope](https://www.sfml-dev.org/contribute.php#general-considerations) and believe the feature fits it"
+                required: true
+              - label: "I am aware and accept that my feature request can also be declined"
+                required: true
+    - type: textarea
+      id: description
+      attributes:
+          label: Describe your feature request here
+          description: A clear and concise description of the wanted change.
+          placeholder: "Tell us what you see! And don't forget to add error message in full"
+      validations:
+          required: true
+    - type: textarea
+      id: use-cases
+      attributes:
+          label: Use Cases
+          description: Explain at least two concrete use cases for this feature.
```

Comment:
[We've always been asking](https://www.sfml-dev.org/contribute.php#requesting-features) for multiple use cases, as it will trigger a deeper thinking process. Instead of a "SFML should do X so I don't have to" you're lead into thinking about who else could benefit from this and does it go beyond my current issue.